graph/appender/sidecards_check.go

```diff
@@ -0,0 +1,47 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/graph/tree"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/services/business/checkers"
+	"strings"
+)
+
+type SidecardsCheckAppender struct {
```

Comment:
Is it Sidecard or Sidecar?  I think it's the envoy Sidecar you are interested in, right?  If I'm wrong just ignore this comment but otherwise please update everywhere.

Reply:
Yes, wrong name. I'll rename in all places.